spec/models/organisation_spec.rb

```diff
@@ -111,45 +111,52 @@
     context "with associated rent periods" do
       let(:organisation) { create(:organisation) }
       let(:period_1_label) { "Every minute" }
+      let(:period_2_label) { "Every decade" }
+      let(:period_3_label) { "Every century" }
+      let(:period_4_label) { "Every millennium" }
       let(:fake_rent_periods) do
         {
           "1" => { "value" => period_1_label },
-          "2" => { "value" => "Every decade" },
+          "2" => { "value" => period_2_label },
+          "3" => { "value" => period_3_label },
+          "4" => { "value" => period_4_label },
         }
       end
 
       before do
-        create(:organisation_rent_period, organisation:, rent_period: 1)
+        [4, 2, 1].each do |rent_period|
+          create(:organisation_rent_period, organisation:, rent_period:)
+        end
         allow(RentPeriod).to receive(:rent_period_mappings).and_return(fake_rent_periods)
       end
 
       context "when the org does not use all rent periods" do
         it "#rent_periods returns the correct ids" do
-          expect(organisation.rent_periods).to eq [1]
+          expect(organisation.rent_periods).to eq [4, 2, 1]
```

Comment:
Is the order guaranteed when returning the ids? It seems like this could fail with re-ordering but we wouldn't actually care for this test, so could maybe test in a way that doesn't care about the order?

Reply:
I'd imagine the order is, as they're created in that order. However, as this test is not to check the order but to check the presence of the correct ids only I've updated two tests to make sure the array contains the values 4,2,1 not for the array to match [4, 2, 1].